README.md

```diff
@@ -1,6 +1,6 @@
 # Bitmovin Player React Native
```

Comment:
In general, what do you think about changing the structure a bit & renaming headings to eg something like this:

## Platform Support
## Feature Support
## Documentation
## Maintenance and Updates
## Contributing to this project
## Raising a feature suggestion
## Reporting a bug
## Support and SLA disclaimer
## Need more help?